modules/machineset-creating.adoc

```diff
@@ -97,16 +97,16 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      sigs.k8s.io/cluster-api-cluster: cluster_name
-      sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
   template:
     metadata:
       creationTimestamp: null
       labels:
-        sigs.k8s.io/cluster-api-cluster: <cluster_name>
-        sigs.k8s.io/cluster-api-machine-role: <machine_label>
-        sigs.k8s.io/cluster-api-machine-type: <machine_label>
-        sigs.k8s.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role:  <machine_label>
+        machine.openshift.io/cluster-api-machine-type: <machine_label>
+        machine.openshift.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
 ...
 ----
 <1> Ensure that both the `sigs.k8s.io/cluster-api-machineset` parameter values
```

Comment:
`sigs.k8s.io/cluster-api-machineset` should be `machine.openshift.io/cluster-api-machineset`